applications/openshift/etcd/etcd_max_wals/oval/shared.xml

```diff
@@ -0,0 +1,30 @@
+<def-group>
+  <definition class="compliance" id="etcd_max_wals" version="1">
+    <metadata>
+      <title>Disable etcd Auto Log Rotation</title>
+      <affected family="unix">
+        <platform>multi_platform_ocp</platform>
+      </affected>
+      <description>Ensure OpenShift is not auto-rotating logs.</description>
+    </metadata>
+    <criteria>
+      <criterion comment="log auto-rotation is disabled" test_ref="test_etcd_max_wals" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="etcd_max_wals is set to 0" id="test_etcd_max_wals" version="1">
+    <ind:object object_ref="object_etcd_max_wals" />
+    <ind:state state_ref="state_etcd_max_wals" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="object_etcd_max_wals" version="1">
+    <ind:filepath>/etc/etcd/etcd.conf</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*ETCD_MAX_WALS=(.*)$</ind:pattern>
+    <ind:instance datatype="int">1</ind:instance>
+  </ind:textfilecontent54_object>
+
+  <ind:textfilecontent54_state id="state_etcd_max_wals" version="1">
+    <ind:subexpression datatype="int" operation="pattern match">0</ind:subexpression>
```

Comment:
```suggestion
    <ind:subexpression datatype="int" operation="equals">0</ind:subexpression>
```

Reply:
didn't catch this earlier. ``int`` datatypes accept ``equals``, ``not equal``, ``greater than``, ``greater than or equal``, ``less than``, ``less than or equal``